src/main/java/duke/utilities/Ui.java

```diff
@@ -0,0 +1,180 @@
+package duke.utilities;
+
+import java.util.Scanner;
+
+import duke.tasks.Task;
+import duke.utilities.TaskList;
+
+/**
+ * UI class that prints all messages that is shown to the user.
+ */
+public class Ui {
+    protected static Scanner sc;
+    protected String br = "____________________________________________________________\n";
```

Comment:
Just a nit, but perhaps br could be final instead since you want it to remain constant?
```suggestion
    protected final String br = "____________________________________________________________\n";
```

Reply:
Yes, final would be more appropriate for this instance. Thank you for your comment. Ultimately decided that there was no need for a line break after the ui had been implemented.